File: CyberSecurity/Java/CEO.java
public class CEO extends SystemAdmin {
    private String companyName;

    public CEO() {
        super();
        this.companyName = "TechCorp";
        this.accessLevel = "Executive";
    }

    public CEO(String name) {
        super(name);
        this.companyName = "TechCorp";
        this.accessLevel = "Executive";
    }

    public CEO(String name, String company) {
        super(name);
        this.companyName = company;
        this.accessLevel = "Executive";
    }

    protected void getCeoInfo(){
        System.out.println("CEO Name: " + name + ", Company: " + companyName + ", Access Level: " + accessLevel);
    }

    public void registerIP(String ip) {
        System.out.println("CEO " + name + " registered IP: " + ip + " with executive privileges");
    }

    public void monitorSystemAdmin(String adminName) {
        System.out.println("CEO " + name + " monitoring SystemAdmin: " + adminName);
    }

    public void monitorEmployee(String empName) {
        System.out.println("CEO " + name + " monitoring employee: " + empName + " at executive level");
    }
}

File: CyberSecurity/Python/ceo.py
from employee import Employee

class CEO(Employee):
    def __init__(self, name, e_id, department):
        super().__init__(name, department, e_id, "2022-01-01")
        self.access_level = "CEO"

    def register_ip(self, ip):
        print(f"CEO {self.name} registered IP: {ip} with CEO privileges")

    def monitor_employee(self, emp_name):
        print(f"CEO {self.name} monitoring employee: {emp_name}")

    def monitor_admin(self, admin_name):
        print(f"CEO {self.name} monitoring SystemAdmin: {admin_name}")
